### PR TITLE
kb(common): Add KB for TelerikRootComponent exception

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: knowledge-base/common-component-requires-telerikrootcomponent.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Telerik Component Requires a TelerikRootComponent
+description: Learn how to fix the exception about a Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component.
```
Comment (yordan-mitev) [inline knowledge-base/common-component-requires-telerikrootcomponent.md:3]:
```suggestion
description: Learn how to fix the exception "А Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component".
```

File: knowledge-base/common-component-requires-telerikrootcomponent.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* A [.NET 8 Blazor Web App](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app) has a `TelerikRootComponent` in a used layout file, but the app's **Interactivity location** is set to **Per page/component**. In such cases, the layout files use static render mode and the `TelerikRootComponent` is unable to pass cascading values down the component tree.
+
+
+## Solution
+
+To fix and avoid the `TelerikRootComponent` exception:
+
+* [Add a `TelerikRootComponent` to one or more layout files]({%slug getting-started/what-you-need%}#telerikrootcomponent), depending on the application's layout structure.
+* When using the .NET 8 Blazor Web App project template, [create the app with a **Global** interactivity location](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#apply-a-render-mode-to-the-entire-app).
+* On the other hand, if your scenario requires a **Per page/component** interactivity location, then place the `TelerikRootComponent` in an interactive Razor component. Refer to [Using `TelerikRootComponent` with Per Component Interactivity]({%slug rootcomponent-percomponent%}).
```
Comment (yordan-mitev) [inline knowledge-base/common-component-requires-telerikrootcomponent.md:50]:
```suggestion
* If your scenario requires a **Per page/component** interactivity location, place the `TelerikRootComponent` in an interactive Razor component. Refer to [Using `TelerikRootComponent` with Per Component Interactivity]({%slug rootcomponent-percomponent%}).
```

**Review by yordan-mitev — APPROVED** [review]

File: knowledge-base/common-component-requires-telerikrootcomponent.md (hunk 1 above)
Comment (dimodi) [reply knowledge-base/common-component-requires-telerikrootcomponent.md:3]: Quotation marks in the page meta information will break the resulting HTML.

File: knowledge-base/common-component-requires-telerikrootcomponent.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
+title: Telerik Component Requires a TelerikRootComponent
+description: Learn how to fix the exception: А Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component.
```
Comment (dimodi) [inline knowledge-base/common-component-requires-telerikrootcomponent.md:3]:
```suggestion
description: Learn how to fix the exception А Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component.
```

**Pull request merged**